DataConnectors/Perimeter81ActivityLogs.json

```diff
@@ -99,5 +99,21 @@
         }
       ]
     }
-  ]
+  ],
+  "metadata": {
+    "id": "1d855d54-0f17-43b3-ad33-93a0ab7b6ce8",
+    "version": "1.0",
+    "kind": "dataConnector",
+    "source": {
+        "kind": "community"
+    },
+    "author": {
+        "name": "Perimeter 81"
+    },
+    "support": {
+        "name": "Perimeter 82",
```

Comment:
The support name doesn't seem to be correct - shouldn't it be Perimeter 81?

Reply:
The value is as mentioned in [Data Connector Support Metadata.xlsx](https://microsoft-my.sharepoint.com/:x:/r/personal/nyassin_microsoft_com/_layouts/15/guestaccess.aspx?e=FfUUgU&CID=F0FE2683-3E74-4584-A675-E62A83121C4E&wdLOR=c360F89BB-01B1-452A-82FB-E7E91CAF248B&share=EVpUqzObGe5BvFGzHwLLvDYBjx7XUFTUkl6jwXixWj2r8A)

Reply:
This needs to be "Perimeter 81" - I think it's a typo in the excel - please fix for correct company name  - there's no Perimeter 82 company.